packages/frontend/src/components/common/Filters/FilterInputs/FilterStringAutoComplete.tsx

```diff
@@ -156,44 +157,48 @@ const FilterStringAutoComplete: FC<Props> = ({
     // memo override component so list doesn't scroll to the top on each click
     const DropdownComponentOverride = useCallback(
         ({ children, ...props }: { children: ReactNode }) => (
-            <ScrollArea {...props}>
-                {refreshedAt && healthData?.hasCacheAutocompleResults ? (
-                    <Tooltip
-                        withinPortal
-                        position="left"
-                        label={`Click here to cache results now`}
-                    >
+            <Stack w="100%" spacing={0}>
+                <ScrollArea {...props}>
+                    {searchedMaxResults ? (
                         <Text
                             color="dimmed"
                             size="xs"
                             px="sm"
                             pt="xs"
                             pb="xxs"
                             bg="white"
-                            pr="xs"
-                            sx={{ cursor: 'pointer' }}
-                            onClick={() => setForceRefresh(true)}
                         >
-                            Results loaded at {refreshedAt.toLocaleString()}
+                            Showing first {MAX_AUTOCOMPLETE_RESULTS} results.{' '}
+                            {search ? 'Continue' : 'Start'} typing...
                         </Text>
-                    </Tooltip>
-                ) : null}
-                {searchedMaxResults ? (
-                    <Text
-                        color="dimmed"
-                        size="xs"
-                        px="sm"
-                        pt="xs"
-                        pb="xxs"
-                        bg="white"
-                    >
-                        Showing first {MAX_AUTOCOMPLETE_RESULTS} results.{' '}
-                        {search ? 'Continue' : 'Start'} typing...
-                    </Text>
-                ) : null}
+                    ) : null}
 
-                {children}
-            </ScrollArea>
+                    {children}
+                </ScrollArea>
+                {healthData?.hasCacheAutocompleResults ? (
+                    <>
+                        <Tooltip
+                            withinPortal
+                            position="left"
+                            label={`Click here to refresh cache filter values`}
+                        >
+                            <Text
+                                color="dimmed"
+                                size="xs"
+                                px="sm"
+                                p="xxs"
+                                sx={{
+                                    cursor: 'pointer',
+                                    borderTop: '1px solid #E5E5E5',
```

Comment:
```suggestion
                                    borderTop: '1px solid ${theme.colors.gray[2]}',
```
and get the theme from sx={(theme) =>...

Reply:
if you end up doing this, apply a style on hover to change the background so it looks more clickable

Reply:
![image](https://github.com/user-attachments/assets/c323b115-ee86-4985-8a7f-92b316362d26)